VERSION

```diff
@@ -0,0 +1 @@
+2.4.1
```

Comment:
weird that 2.4.0 was our last release to rubygems, but 2.2.0 is our last tag. that's fine, nothing to change here. just pointing out this is correct despite the tags.